packages/@okta/vuepress-site/docs/release-notes/2024-okta-identity-engine/index.md

```diff
@@ -7,6 +7,20 @@ title: Okta Identity Engine API release notes 2024
 
 ## August
 
+### Weekly release 2024.08.3
+
+| Change | Expected in Preview Orgs |
+|--------|--------------------------|
+| [Bugs fixed in 2024.08.3](#bugs-fixed-in-2024-08-3)| August 28, 2024 |
+
+#### Bugs fixed in 2024.08.3
+
+* The system log displayed an incorrect actor when events were triggered by a [Reset factor](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/UserLifecycle/#tag/UserLifecycle/operation/resetFactors) API request. (OKTA-752183)
```

Comment:
thought we were remove links?

Reply:
You said it was optional to remove them so I left the ones which were already included in the blurbs.